code/drasil-printers/lib/Language/Drasil/HTML/Print.hs

```diff
@@ -39,11 +39,16 @@ import Language.Drasil.Printing.Citation (CiteField(Year, Number, Volume, Title,
   Citation(Cite), BibRef)
 import Language.Drasil.Printing.LayoutObj (Document(Document), LayoutObj(..), Tags)
 import Language.Drasil.Printing.Helpers (comm, dot, paren, sufxer, sufxPrint)
-import Language.Drasil.Printing.PrintingInformation (PrintingInformation)
+import Language.Drasil.Printing.Import.Helpers
+  (lookupT, lookupS, lookupP)
```

Comment:
Unless our standards have changed, we allow for a maximum of 80-character-long lines; if something _can_ fit comfortably on one line, might as well, and might as well make the imports alphabetical for ease of finding them later (unless there's a specific reason for this ordering).

```suggestion
import Language.Drasil.Printing.Import.Helpers (lookupP, lookupS, lookupT)
```